src/ert/dark_storage/enkf.py

```diff
@@ -5,20 +5,46 @@
 
 from fastapi import Depends
 
+from ert.config import ErtConfig
 from ert.dark_storage.security import security
-from ert.storage import Storage, open_storage
+from ert.enkf_main import EnKFMain
+from ert.libres_facade import LibresFacade
+from ert.storage import StorageReader, open_storage
 
-__all__ = ["get_storage"]
+__all__ = ["LibresFacade", "get_res", "get_storage"]
 
-
-_storage: Optional[Storage] = None
+_libres_facade: Optional[LibresFacade] = None
+_ert: Optional[EnKFMain] = None
+_config: Optional[ErtConfig] = None
+_storage: Optional[StorageReader] = None
 
 DEFAULT_SECURITY = Depends(security)
 
 
-def get_storage() -> Storage:
+def init_facade() -> LibresFacade:
+    global _libres_facade  # noqa: PLW0603
+    global _config  # noqa: PLW0603
+
+    configfile = os.environ["ERT_STORAGE_RES_CONFIG"]
+
+    _config = ErtConfig.from_file(configfile)
+    os.chdir(_config.config_path)
+    _libres_facade = LibresFacade(_config)
+    return _libres_facade
+
+
+def get_res(*, _: None = DEFAULT_SECURITY) -> LibresFacade:
+    if _libres_facade is None:
+        return init_facade()
+    return _libres_facade
+
+
+DEFAULT_LIBRESFACADE = Depends(get_res)
+
+
+def get_storage(*, res: LibresFacade = DEFAULT_LIBRESFACADE) -> StorageReader:
```

Comment:
It's not a default value. It incorrectly moved out due to an overzealous commitment to listening to pylint (never listen to pylint): https://github.com/equinor/ert/commit/64ed9833a2c8b4f7f001a0c6050f2d9cc5dcf4da

```suggestion
def get_storage(*, res: LibresFacade = Depends(get_res)) -> StorageReader:
```

or the more "modern"

```suggestion
def get_storage(*, res: Annotated[LibresFacade, Depends(get_res)]) -> StorageReader:
```

Reply:
This has been removed as it was not needed for the final PR.